.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,8 +1,5 @@
 blank_issues_enabled: false
 contact_links:
   - name: Auth0 Community
-    url: https://community.auth0.com/c/sdks
+    url: https://community.auth0.com
     about: Discuss this SDK in the Auth0 Community forums
-  - name: Library Documentation
-    url: https://github.com/auth0/jwks-rsa-java/blob/master/README.md
-    about: Read the library documentation
```

Comment:
Since the introduction of EXAMPLES.md the Readme no longer contains all the documentation for this library (not even a significant portion).